util/populate_dev_data.go

```diff
@@ -41,48 +41,64 @@ func main() {
 		log.Fatal(err)
 	}
 
-	emptySecretToken := &base.Token{}
+	emptySecretToken := &shared.Token{}
 	staticDataTime, _ := time.Parse(time.RFC3339, "2017-10-18T00:00:00Z")
 
 	// Follow pattern established in run/*.py data collection code.
 	const staticRunSHA = "b952881825"
 	const summaryURLFmtString = "/static/" + staticRunSHA + "/%s"
-	staticTestRuns := []base.TestRun{
+	staticTestRuns := []shared.TestRun{
 		{
-			BrowserName:    "chrome",
-			BrowserVersion: "63.0",
-			OSName:         "linux",
-			OSVersion:      "3.16",
-			Revision:       staticRunSHA,
-			ResultsURL:     fmt.Sprintf(summaryURLFmtString, "chrome-63.0-linux-summary.json.gz"),
-			CreatedAt:      staticDataTime,
+			ProductAtRevision: shared.ProductAtRevision{
+				Product: shared.Product{
+					BrowserName:    "chrome",
+					BrowserVersion: "63.0",
```

Comment:
Is this what the version number looks like in prod? Would be good to test with real version numbers, but only when prod has real ones.

Reply:
Not any more - now they're 63.0.9999.123456 or similar. Will follow up with the sem-ver partial-matching in another PR.